lsst-epo/investigations-client
# [C] menu content

JIRA: https://jira.lsstcorp.org/browse/EPO-8670

## What this change does ##

Fleshes out menu content with acknowledgements, sharing, and help. See corresponding API branch.

## Notes for reviewers ##

Review this change on both the client and API. 

Run `yarn` to pick up latest version of libs that makes updates to slideout menu and sharing buttons

In Craft a new "Help URL" should be available in the globals, and under Investigation parent entries an acknowledgements field should be available.

Include an indication of how detailed a review you want on a 1-10 scale.
-5 

## Testing ##

- Add test acknowledgements to Coloring the Universe in Craft
- Add a test help URL to the globals in Craft
- Open the menu in an investigation and verify acknowledgements
- Open the menu in an investigation and verify help link
- Open the menu in an investigation and verify sharing

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##
### Before:
### After:
